src/components/AudioPlayer/AudioPlayer.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+
+import AudioPlayer from "./AudioPlayer";
+
+const meta: Meta<typeof AudioPlayer> = {
+  title: "Components/Media & Icons/AudioPlayer",
+  component: AudioPlayer,
+  decorators: [withDesign],
+  argTypes: {
+    audioType: {},
```

Comment:
Can you make this control a radio type with the four string types? That's how the current Controls displays this prop and the set of four values.